### DISCUSSION TITLE
The cypress npm package is installed, but the Cypress binary is missing.

### DISCUSSION THREAD

**Discussion by kalpit1981** [cypress-io/cypress#27780] [CI setup]
I want to configure a release pipeline in Azure DevOps to run my Cypress test scripts. When manually triggering the release pipeline, I always get the error: **The cypress npm package is installed, but the Cypress binary is missing.** (Kindly refer to the screenshots attached, where I have given my package.json file, how I have set my release pipeline and npm custom script where it is failing. )
Can someone explain the issue and also give a solution in detail that would be great. Thank You. I have gone through a few threads with similar kinds of issues but am unable to understand them completely and that's why asking for help.

![JsonFile](https://github.com/cypress-io/cypress/assets/144344781/d664b2a8-c79c-4ea2-88ed-793ce934d8a6)
![Pipeline Logs](https://github.com/cypress-io/cypress/assets/144344781/0604da5a-3a5f-4c62-a5ea-d9bb36cac575)
![Pipeline1](https://github.com/cypress-io/cypress/assets/144344781/a02fdba6-9415-460e-aaef-70aa50976adf)
![Pipeline2](https://github.com/cypress-io/cypress/assets/144344781/c310011f-3c5b-4085-8625-30e1264dfe34)
![Pipeline3](https://github.com/cypress-io/cypress/assets/144344781/dbdd5850-8382-49d4-8a5f-9a69c893b569)

**Comment by JakeShawRicardo** [comment]
Did you ever resolve this issue? I had a working project but after deleting and reinstalling the node modules folder I am having the same error.
